examples/scene/instanced_mesh.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Instanced rendering of arbitrarily transformed meshes
+=====================================================
+"""
```

Comment:
It looks like this example isn't getting a proper thumbnail/image extracted from it. Do you want to try adding a `gallery` line and see if the doc rendering can generate an image?

https://vispy.org/dev_guide/writing_examples.html#specify-frames-to-grab